Sources/DeepLearning/MNISTdataset.swift

```diff
@@ -0,0 +1,91 @@
+import Python
+import TensorFlow
+
+let np = Python.import("numpy")
+let path = Python.import("os.path")
+let urllib = Python.import("urllib.request")
+let gzip = Python.import("gzip")
+let sh = Python.import("shutil")
+
+let URL = "http://yann.lecun.com/exdb/mnist/"
```

Comment:
To avoid polluting the namespace, could you define these constants within an object?

Reply:
Hey @saeta , I have made all changes as you requested.